Abiodun248/myDiaryApp
# #174300531 Setup the server side of the application using the specified framework

#### What does this PR do? 
It create a server for the app

#### Description of Task to be completed?
i installed express,
created an endpoint to get the server,
i installed mocha and chai (and chaiHttp) to write test

#### How should this be manually tested?
after cloning the app, cd into it and run app.js.
then check with postman by this url: "http://localhost:5000"

#### Any background context you want to provide?
There is no context

#### What are the relevant pivotal tracker stories?
#174300531

#### Screenshots (if appropriate)
#### Questions:
